src/DeviceRow.vala

```diff
@@ -239,27 +270,43 @@ public class Bluetooth.DeviceRow : Gtk.ListBoxRow {
                 if (settings_button.uri != "") {
                     settings_button.visible = true;
                 }
+                forget_button.sensitive = true;
+                forget_button.visible = true;
                 break;
             case Status.CONNECTING:
                 connect_button.sensitive = false;
                 state.icon_name = "user-away";
                 settings_button.visible = false;
+                forget_button.sensitive = false;
+                forget_button.visible = true;
                 break;
             case Status.DISCONNECTING:
                 connect_button.sensitive = false;
                 state.icon_name = "user-away";
                 settings_button.visible = false;
+                forget_button.sensitive = false;
+                forget_button.visible = true;
                 break;
             case Status.NOT_CONNECTED:
                 connect_button.label = _("Connect");
                 connect_button.sensitive = true;
                 state.icon_name = "user-offline";
                 settings_button.visible = false;
+                forget_button.sensitive = true;
+                forget_button.visible = true;
                 break;
             case Status.UNABLE_TO_CONNECT:
                 connect_button.sensitive = true;
                 state.icon_name = "user-busy";
                 settings_button.visible = false;
+                forget_button.visible = false;
+                break;
+            case Status.UNABLE_TO_CONNECT_PAIRED:
```

Comment:
Is there a difference between this and `NOT_CONNECTED`?

Reply:
`NOT_CONNECT` is used to indicate when a bluetooth fail to pair: on this case we do not want to show the `forget` button, so I added: `NOT_CONNECT_PAIRED` to indicate the state when a device fail to connect but was paired, on this case we want to show de `forget` button.